stock_secondary_unit_move_line/models/stock_picking.py

```diff
@@ -0,0 +1,18 @@
+# © 2025 Solvos Consultoría Informática (<http://www.solvos.es>)
+# License AGPL-3 - See http://www.gnu.org/licenses/agpl-3.0.html
+
+from odoo import models
+
+
+class StockPicking(models.Model):
+    _inherit = ["stock.picking", "product.secondary.unit.mixin"]
+    _name = "stock.picking"
+
+    def _action_done(self):
+        res = super()._action_done()
+
+        for picking in self:
+            if picking.backorder_ids:
+                for line in picking.backorder_ids.move_ids:
+                    line._compute_secondary_uom_qty()
```

Comment:
compute method should be multi, so it could be called in that way, please try:
```suggestion
                picking.backorder_ids.move_ids._compute_secondary_uom_qty()
```